smokey/src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -0,0 +1,55 @@
+// ************************************************************
```

Comment:
this isn't the latest from Veer. Is there a reason not to start from the latest on Veer and modify from there?

Reply:
again ... I got this from Veer in main branch ... the other code must be off in some other feature branch

Reply:
Found some stuff out in the swerve-shuffleboard-to-navX branch that hopefully is closer.  We need to have folks finish their work and get it to main going forward.

Reply:
What's on the main branch of veer looks right
https://github.com/Team4682CyBears/ChargedUp2023/blob/main/veer/src/main/java/frc/robot/commands/DefaultDriveCommand.java

But that's not what this file has.  For example, line 22 should have a boolean flag to toggle field oriented drive... 
https://github.com/Team4682CyBears/ChargedUp2023/blob/5a3f202910f254a7ca19229bdff7759d9ae728b8/veer/src/main/java/frc/robot/commands/DefaultDriveCommand.java#L22